PaddleAudio/paddleaudio/models/wav2vec2/tokenizer.py

```diff
@@ -0,0 +1,398 @@
+import json
+from itertools import groupby
+from typing import Dict, List, Optional, Tuple, Union
+
+from paddle.utils import download
+from paddleaudio.utils.log import Logger
+
+URL_BASE = 'https://bj.bcebos.com/paddleaudio/models/wav2vec2/'
+logger = Logger(__file__)
+
+
+class Wav2Vec2Tokenizer():
+    """
+    Constructs a Wav2Vec2 tokenizer.
+
+    This tokenizer inherits from :class:`~transformers.PreTrainedTokenizer` which contains some of the main methods.
```

Comment:
You need to update the comments according to our situation.

Reply:
comments are fixed now.